network/wsPeer.go

```diff
@@ -48,15 +49,10 @@ const averageMessageLength = 2 * 1024    // Most of the messages are smaller tha
 const msgsInReadBufferPerPeer = 10
 
 var networkSentBytesTotal = metrics.MakeCounter(metrics.NetworkSentBytesTotal)
-var networkSentBytesByTag = metrics.NewTagCounter("algod_network_sent_bytes_{TAG}", "Number of bytes that were sent over the network per message tag")
```

Comment:
why does your change requires the removal of the metrics ?

Reply:
@yossigi, hey was there a reason these were removed? If not, I will add them back in the right locations.